src/DotVVM.Samples.Common/DotvvmStartup.cs

```diff
@@ -94,6 +94,7 @@ private static void AddControls(DotvvmConfiguration config)
             config.Markup.AddMarkupControl("sample", "Localization_Control", "Views/FeatureSamples/Localization/Localization_Control.dotcontrol");
             config.Markup.AddMarkupControl("sample", "ControlCommandBinding", "Views/FeatureSamples/MarkupControl/ControlCommandBinding.dotcontrol");
             config.Markup.AddMarkupControl("sample", "ControlValueBindingWithCommand", "Views/FeatureSamples/MarkupControl/ControlValueBindingWithCommand.dotcontrol");
+            config.Markup.AddMarkupControl("sample", "EmbeddedResourceControlsButton", "Views/FeatureSamples/EmbeddedResourceControls/MyButton.dotcontrol");
```

Comment:
This actually loads the control from the filesystem. The purpose of this feature is to load the control from the embedded resource.